.travis.yml

```diff
@@ -7,18 +7,30 @@ install:
 
 jobs:
   include:
-    - name: "Build, Test, and Run Coverage"
+    - name: "Build, Test, and Run Coverage (All Except SimulatedTests)"
       script:
         - cd src
         # Run tests and collect coverage
         - >
-          bazel coverage //...     \
-            --compilation_mode=fastbuild    \
-            --verbose_test_summary          \
-            --instrumentation_filter=//...  \
-            --test_output=errors
+          bazel coverage                     \
+            --compilation_mode=fastbuild     \
+            --verbose_test_summary           \
+            --instrumentation_filter=//...   \
+            --test_output=errors             \
+            -- //... -//software/simulated_tests/...
         # Collect Code Coverage Results
-        - bash <(curl -s https://codecov.io/bash) -s bazel-testlogs/ | (head -n100 && tail -n100)
+        # Only print the first 100 lines of the output of the code coverage upload script
+        # so we don't overflow the Travis CI logs
+        - bash <(curl -s https://codecov.io/bash) -s bazel-testlogs/ | head -n100
```

Comment:
I would really recommend both first and last 100 lines, since errors are usually at the end  or the start. After some googling, the easiest option seems to be to write the output to a file and then all `head` and `tail` on that file at the end.